packages/core-app-api/config.d.ts

```diff
@@ -65,6 +65,12 @@ export interface Config {
         }>;
       }>;
     };
+
+    /**
+     * An optional sign out url that can be used with oauth2 proxy
+     * @visibility frontend
+     */
+    signOutUrl?: string;
```

Comment:
Should this be under auth instead of app perhaps? 🤔

Reply:
I went back and forth on this one. Ultimately I wanted to place this under the proxy portion of auth, however since it was not defined in the config schema for `core-app-api`, I didn't want to cause confusion with users having the two be separated under the auth section. I also ended up choosing the app section as to keep the `signOutUrl` with the `baseUrl`. But I am definitely open to suggestions on this on.